diff --git a/bin/nest_vars.sh.in b/bin/nest_vars.sh.in
index 6e33190cd..37429f5c5 100644
--- a/bin/nest_vars.sh.in
+++ b/bin/nest_vars.sh.in
@@ -1,7 +1,13 @@
 #!/bin/sh

+# `site.USER_SITE` is for installation with --user
+
+# Make SLI available
+export SLI_PATH="$( python3 -c 'import site; print(site.USER_SITE)' )"
+
 # Make PyNEST available by prepending its path to PYTHONPATH in a safe way.
-export PYTHONPATH="@CMAKE_INSTALL_PREFIX@/@PYEXECDIR@${PYTHONPATH:+:$PYTHONPATH}"
+export PYTHONPATH="$( python3 -c 'import site; print(site.USER_SITE)' )${PYTHONPATH:+:$PYTHONPATH}"

 # Make NEST executables available by prepending their path to PATH.
-export PATH="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR@:${PATH}"
+export PATH="$( python3 -c 'import site; print(site.USER_SITE)' )/bin:${PATH}"
+
